data/benefits/javascript/bretagne-jeune-international.yml

```
@@ -16,9 +16,21 @@ conditions_generales:
     operator: "<="
 profils:
   - type: enseignement_superieur
-link: https://www.bretagne.bzh/aides/fiches/bourse-jeunes-a-linternational-etudiants-des-universites/
+    conditions:
+      - type: annee_etude
+        values:
+          - but_1
+          - licence_1
+          - licence_2
+          - licence_3
+          - master_1
+          - master_2
+link: https://www.bretagne.bzh/actualites/bougez-en-europe-et-dans-le-monde/
 teleservice: https://extranets.region-bretagne.fr/Portail-Aides/jsp/nouveauContexte.action?codeAction=M42-CONNEXION
 prefix: la
-type: bool
-periodicite: ponctuelle
+type: float
+unit: €
+periodicite: mensuelle
+legend: maximum
+montant: 300
```

Me semble que c'est maximum 200€ par mois ici :
```suggestion
montant: 200
```